### PR TITLE
Update WASM documentation

### PR DESCRIPTION
**Pull request by aepanchi**
### Description 
Update the file with build and test instructions. 

Fixes # - _issue number(s) if exists_

- [x] - git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/oneapi-src/oneTBB/blob/master/CONTRIBUTING.md#pull-requests) for details)_

### Type of change

_Choose one or multiple, leave empty if none of the other choices apply_

_Add a respective label(s) to PR if you have permissions_

- [ ] bug fix - _change that fixes an issue_
- [ ] new feature - _change that adds functionality_
- [ ] tests - _change in tests_
- [ ] infrastructure - _change in infrastructure and CI_
- [x] documentation - _documentation update_

### Tests

- [ ] added - _required for new features and some bug fixes_
- [x] not needed

### Documentation

- [ ] updated in # - _add PR number_
- [ ] needs to be updated
- [x] not needed

### Breaks backward compatibility
- [ ] Yes
- [x] No
- [ ] Unknown

### Notify the following users
_List users with `@` to send notifications_

### Other information
Signed-off-by: Alexandra Epanchinzeva alexandra.epanchinzeva@intel.com

### CODE REVIEW

File: WASM_Support.md [hunk 1]
```diff
@@ -16,16 +16,57 @@
 
 # WASM Support
 
+oneTBB extends its capabilities by offering robust support for ``WASM``. 
+
 ``WASM`` stands for WebAssembly, a low-level binary format for executing code in web browsers. 
-It is designed to be a portable target for compilers and to be efficient to parse and execute. 
+It is designed to be a portable target for compilers and efficient to parse and execute. 
+
+By integrating oneTBB with WASM, you can take full advantage of parallelism and concurrency while working on web-based applications, interactive websites, and a variety of other WASM-compatible platforms.
```
Comment (pavelkumbrasev) [inline WASM_Support.md:24]: I don't understanding the meaning of:
'By integrating oneTBB with WASM, you can take full advantage'
Who will integrate what with what?

'oneTBB offers WASM support through the integration with'
Means that oneTBB provides integration with WASM.

File: WASM_Support.md [hunk 2]
```diff
@@ -16,16 +16,57 @@
...
-* Take advantage of oneTBB features to optimize the performance of your web-based applications.
+em++ -Wno-unused-command-line-argument -pthread -s TOTAL_MEMORY=128MB -s EXIT_RUNTIME=1 -s PROXY_TO_PTHREAD=1 -O2 -g -DNDEBUG -pthread -rdynamic <test_file>.o -o <test_file>.js libtbb.a -ldl
 
+node --experimental-wasm-threads --experimental-wasm-bulk-memory <test_file>.js
+```
 
+Where: 
 
+* ``--experimental-wasm-threads`` - enables experimental support for WASM threads in *Node.js*.
+* ``--experimental-wasm-bulk-memory`` - enables experimental support for bulk memory operations in *Node.js*.
```
Comment (pavelkumbrasev) [inline WASM_Support.md:72]: Do we really need it here?

**Review by pavelkumbrasev — COMMENTED** [review]
@JhaShweta1 Could you please take a look?

File: WASM_Support.md (hunk 2 above)
Comment (JhaShweta1) [reply WASM_Support.md:72]: I think we can remove line 59 to 72. Command 'ctests' works, so these instructions aren't needed.

File: WASM_Support.md (hunk 1 above)
Comment (aepanchi) [reply WASM_Support.md:24]: @pavelkumbrasev 
1) maybe rewrite it like that: Using oneTBB with WASM, you can......
2) should be like that: oneTBB provides integration with WASM using Emscripten* ....

Comment (pavelkumbrasev) [reply WASM_Support.md:24]: Let's apply the first one: `Using oneTBB with WASM, you can......`
And keep the second as it was originally

**Review by pavelkumbrasev — APPROVED** [review]

**Pull request merged**